contrib/build-push-ceph-container-imgs.sh

```diff
@@ -6,15 +6,28 @@ set -ex
 #############
 # VARIABLES #
 #############
+function require {
+  if [[ -z "${!1}" ]] ; then
+    echo "Required variable $1 not set, exiting"
+    exit 1
+  fi
+}
+
 CI_CONTAINER=${CI_CONTAINER:=false}
 if ${CI_CONTAINER} ; then
   # save the ceph branch, from the parent job
   CEPH_BRANCH=${BRANCH}
   # must be set by caller (perhaps another Jenkins build job)
+  # BRANCH (as above, the Ceph branch)
+  # SHA1 (sha1 corresponding to the Ceph branch)
   # CONTAINER_REPO_HOSTNAME="quay.io"
   # CONTAINER_REPO_ORGANIZATION="cephci"
   # CONTAINER_REPO_USERNAME=user
   # CONTAINER_REPO_PASSWORD=password
+  for v in BRANCH SHA1 CONTAINER_REPO_HOSTNAME CONTAINER_REPO_ORGANIZATION \
```

Comment:
If the list might change we could make it an array and iterate over it, if not let's keep it this way.